### ISSUE TITLE
does facenet support docker

### ISSUE THREAD

**Issue by upgradepa** [davidsandberg/facenet#1231]

**Comment by Jawabreh0** [comment]
Yes, it can be run in a Docker container. There are several Docker images available that provide a pre-configured environment for running FaceNet, such as this one: https://hub.docker.com/r/jfloff/keras-notebook/.

To run FaceNet in a Docker container, you would first need to install Docker on your machine. Once Docker is installed, you can pull the FaceNet image you want to use from Docker Hub or build your own custom image with the necessary dependencies and configurations. Then, you can run the FaceNet container and execute the desired commands.
